components/org.wso2.carbon.identity.api.server.userstore/org.wso2.carbon.identity.api.server.userstore.v1/src/main/java/org/wso2/carbon/identity/api/server/userstore/v1/core/ServerUserStoreService.java

```diff
@@ -447,11 +450,37 @@ private UserStoreDTO createUserStoreDTO(UserStoreReq userStoreReq) {
             userStoreList.setSelf(ContextLoader.buildURIForBody(String.format(V1_API_PATH_COMPONENT +
                             UserStoreConstants.USER_STORE_PATH_COMPONENT + "/%s",
                     base64URLEncodeId(jsonObject.getDomainId()))).toString());
+
+            if (StringUtils.isNotBlank(requiredAttributes)) {
+                String[] requiredAttributesArray = requiredAttributes.split(",");
```

Comment:
Shall we move this separator to a constant file?

Reply:
fixed